t/plugin/udp-logger.t

```diff
@@ -431,8 +405,29 @@ the mock backend is hit
                 ngx.say(body)
                 return
             end
-
+            local code, body = t('/apisix/admin/plugin_metadata/udp-logger',
+                 ngx.HTTP_PUT,
+                 [[{
+                        "log_format": {
+                            "host": "$host",
+                            "case name": "logger format in plugin",
+                            "@timestamp": "$time_iso8601",
+                            "client_ip": "$remote_addr"
+                        }
+                }]]
+                )
+            if code >= 300 then
+                ngx.status = code
+                ngx.say(body)
+                return
+            end
             ngx.say(body)
+            local code, _, _ = t("/hello", "GET")
```

Comment:
need a blank before this block

Reply:
oops I had missed it. Added now. @monkeyDluffy6017